app/src/screens/MapScreen.js

```diff
@@ -74,15 +78,15 @@ function MapScreen(): React.Node {
   const [unfilteredLayers, setUnfilteredLayers] = useState<LayerDict>({});
   const [filteredLayers, setFilteredLayers] = useState<LayerDict>({});
   const [hoverInfo, setHoverInfo] = useState<?PickInfo<Point>>(null);
-  const [minRssiToDisplay, setMinRssiToDisplay] = useState<number>(-1000);
-  const [maxRssiToDisplay, setMaxRssiToDisplay] = useState<number>(0);
   const [mapStyle, setMapStyle] = useState<string>(
     'mapbox://styles/mapbox/satellite-v9',
   );
   const [satelliteView, setSatelliteView] = useState<boolean>(true);
   const [cacheMapDialogOpen, setCacheMapDialogOpen] = useState<boolean>(false);
   const [filterMinRssi, setFilterMinRssi] = useState('');
   const [filterMaxRssi, setFilterMaxRssi] = useState('');
+  const [filterMinHeight, setFilterMinHeight] = useState('10');
+  const [filterMaxHeight, setFilterMaxHeight] = useState('');
```

Comment:
Just a nit: instead of 'height' maybe use 'elevation' or 'altitude' instead.

Reply:
yea, I used that for the UI text but height was shorter, and I chose that first LOL. I can change it.